### ISSUE TITLE
Issue detected on the blog-stats as well on the project-stats with icons align 

### ISSUE THREAD

**Issue by therockstarind** [ccrsxx/portofolio#100]
Replace to this 
```tsx
import { HiEye, HiClock } from 'react-icons/hi2';
import { Accent } from '@components/ui/accent';
import { ViewsCounter } from '@components/content/views-counter';
import type { Blog } from '@lib/types/contents';
import type { PropsForViews } from '@lib/types/helper';

type BlogStatProps = PropsForViews<Pick<Blog, 'readTime'>>;

export function BlogStats({
  slug,
  readTime,
  increment
}: BlogStatProps): JSX.Element {
  return (
    <div className='flex gap-4 text-sm font-medium text-gray-600 dark:text-gray-300'>
      <div className='flex items-center gap-1'>
      <HiClock className='h-4 w-4' />
        <Accent>{readTime}</Accent>
      </div>
      <div className='flex items-center gap-1'>
      <HiEye className='h-4 w-4' />
        <Accent>
          <ViewsCounter slug={slug} increment={increment} />
        </Accent>
      </div>
    </div>
  );
}
```
on this component

[src/components/blog/blog-stats.tsx](https://github.com/ccrsxx/portofolio/blob/main/src/components/blog/blog-stats.tsx)

also replace this 
```tsx
import { HiEye, HiUser, HiClock, HiLink } from 'react-icons/hi2';
import { SiGithub, SiYoutube } from 'react-icons/si';
import { CustomLink } from '@components/link/custom-link';
import { ViewsCounter } from '@components/content/views-counter';
import type { IconType } from 'react-icons';
import type { Project } from '@lib/types/contents';
import type { PropsForViews } from '@lib/types/helper';

type ProjectLinks = {
  name: string;
  link: string | undefined;
  Icon: IconType;
};

type ProjectStatsProps = PropsForViews<
  Pick<Project, 'readTime' | 'link' | 'category' | 'github' | 'youtube'>
>;

export function ProjectStats({
  slug,
  link,
  github,
  youtube,
  readTime,
  category,
  increment
}: ProjectStatsProps): JSX.Element {
  const projectLinks: ProjectLinks[] = [
    {
      name: 'Repository',
      link: github,
      Icon: SiGithub
    },
    {
      name: 'Demo',
      link: youtube,
      Icon: SiYoutube
    },
    {
      name: 'Open Live Site',
      link: link,
      Icon: HiLink
    }
  ];

  return (
    <>
      <div className='flex flex-wrap gap-x-4 gap-y-2 text-sm font-medium text-gray-600 dark:text-gray-300'>
        <div className='flex items-center gap-1'>
          <HiClock className='h-4 w-4' />
          <p>{readTime}</p>
        </div>
        <div className='flex items-center gap-1'>
        <HiEye className='h-4 w-4' />
          <ViewsCounter slug={slug} increment={increment} />
        </div>
        {projectLinks.map(
          ({ name, link, Icon }) =>
            link && (
              <div className='flex items-center gap-3' key={name}>
                <i>
                  <Icon className='text-lg text-black dark:text-white' />
                </i>
                <CustomLink href={link}>{name}</CustomLink>
              </div>
            )
        )}
      </div>
      <div className='flex items-center gap-2 text-gray-600 dark:text-gray-300'>
        <i>
          <HiUser className='text-lg' />
        </i>
        <p className='text-sm'>{category}</p>
      </div>
    </>
  );
}

```
on this component
[src/components/project/project-stats.tsx](https://github.com/ccrsxx/portofolio/blob/main/src/components/project/project-stats.tsx)

**Comment by ccrsxx** [comment]
Looks fine to me, it's correctly aligned on my end. I used `flex items-center` on the container that contains the icon and the text, it should be correctly aligned.

Can you be more specific on what is the problem?

**Comment by therockstarind** [comment]
haha 😄 I'm not talking about the center on view prop you use time icon on time prop you use eye icon ![Screenshot_20231129_213205_Brave.jpg](https://github.com/ccrsxx/portofolio/assets/45270927/4d9d715d-3071-42b1-bdb6-5c88711bd805)

check in the image

**Comment by ccrsxx** [comment]
Fixed it on commit b2721b5.
